src/window.rs

```diff
@@ -596,8 +596,15 @@ impl AppWindow {
 
     /// Update the current color to the given color.
     /// The new color will be added to the history list.
+    /// Previous entries of the same color will be deleted.
```

Comment:
```suggestion
    /// Set the current color to the given color.
    /// 
    /// If the given color is different from the current color, 
    /// it will be added to the history. If the history includes the given 
    /// color, the preceding occurrence will be removed.
```